scripts/main.js

```diff
@@ -1,6 +1,16 @@
-const { getReposMoreThanFiveStars: getReposMoreThanFiveStars } = require("./functions");
-const { getLastUpdatedRepos: getLastUpdatedRepos } = require("./functions");
+const {
+  getReposMoreThanFiveStars,
+  getLastUpdatedRepos,
+  getSumOfReposStars,
+  getArrayOfData,
+} = require("./functionsAsyncAwait");
 
-getReposMoreThanFiveStars().then(data => console.log(data));
-
-getLastUpdatedRepos().then(data => console.log(data));
\ No newline at end of file
+getArrayOfData()
+  .then((data) => {
+    console.log(getReposMoreThanFiveStars(data));
+    console.log(getLastUpdatedRepos(data));
+    console.log(getSumOfReposStars(data));
+  })
+  .catch((error) => {
+    console.log(error);
+  });
```

Comment:
You can use currying here!
```suggestion
  .catch(console.log);
```